dashboard/src/components/ChevronRightAnimate/ChevronRightAnimate.tsx

```diff
@@ -0,0 +1,9 @@
+import { MdChevronRight } from 'react-icons/md';
+
+const ChevronRightAnimate = (): JSX.Element => {
```

Comment:
Because of the `group`,  this component only makes sense inside the `dashboard/src/components/Accordion/Accordion.tsx`.

This won't be animated anywhere else in the code. 

So, just move there, no need export it or create these new files.